api/v1beta1/topology.go

```diff
@@ -61,3 +62,33 @@ func LinkKuadrantToAuthorino(objs controller.Store) machinery.LinkFunc {
 		},
 	}
 }
+
+func LinkKuadrantToServiceMonitor(objs controller.Store) machinery.LinkFunc {
+	kuadrants := lo.Map(objs.FilterByGroupKind(KuadrantGroupKind), controller.ObjectAs[machinery.Object])
+
+	return machinery.LinkFunc{
+		From: KuadrantGroupKind,
+		To:   schema.GroupKind{Group: monitoringv1.SchemeGroupVersion.Group, Kind: monitoringv1.ServiceMonitorsKind},
+		Func: func(child machinery.Object) []machinery.Object {
+			return lo.Filter(kuadrants, func(kuadrant machinery.Object, _ int) bool {
+				// ServiceMonitors outside the Kuadrant namespace will be outside the topology tree
```

Comment:
Did you consider filtering by label instead of namespace? The operator might not be installed where the kuadrant CR is being created. For instance in Openshift, operators (by default) go to `openshift-operator` whereas the kuadrant CR is usually created in `kuadrant-system`